src/features/helm-charts/child-features/preferences/renderer/adding-of-custom-helm-repository/custom-helm-repo.injectable.ts

```diff
@@ -4,11 +4,12 @@
  */
 import { getInjectable, lifecycleEnum } from "@ogre-tools/injectable";
 import { observable } from "mobx";
+import type { HelmRepo } from "../../../../../../common/helm/helm-repo";
 
 const customHelmRepoInjectable = getInjectable({
   id: "custom-helm-repo",
 
-  instantiate: () => observable({
+  instantiate: () => observable.object<HelmRepo>({
```

Comment:
Is this hack required because of `injectable`? Does it mess up typings of transients?

Reply:
1. I wanted to change the field `cacheFilePath` to be required instead of optional but I noticed that this injectable wasn't typed as such so the type error was in the wrong place.
2. The typings of `observable` are wonky and just specifying a generic param (such as the code `observable<HelmRepo>(...)` types as `IObservableArray<HelmRepo>` for some reason.